src/styles/views/UserPage.scss

```diff
@@ -28,20 +28,26 @@
   &.player-info-container{
     color: $textColor;
     margin-top: 2em;
+    font-size: 30px;
   }
 
   &.button-container {
     position: relative;
-    bottom: 100px;
+    bottom: 120px;
     display: flex;
     justify-content: center;
     margin-top: 2em;
     background-color: transparent;
+    width: 400px;
   }
 
   &.button{
-    background-color: $background;
-
+    background-color: $accent;
+    width: 400px;
```

Comment:
I think this line should be ` width: 400px;` in order to be the same size as #72